### ISSUE TITLE
Actividad: Dive Into Rust

### ISSUE THREAD

**Issue by daviddagb2** [mozillahispano/meta#6]
## Objetivo

Apoyo a la actividad "[Dive Into Rust](https://activate.mozilla.community/developer-engagement/rust-hack)"
## Éxito

30 programadores / desarrolladores se introducen en Rust
10 Artículos sobre eventos u otras actividades locales Rust
15 Mozillians organizando actividades y eventos relacionados a Rust
15 nuevas personas se inscriben en la lista de desarrolladores Rust
## Tareas relacionadas y enlaces

https://users.rust-lang.org/top/all
https://activate.mozilla.community/developer-engagement/rust-hack
## Roles
- Coordinador: @daviddagb2 (David Blanchard)
- Involucrados: @Izeledon
## Requisitos:
- [ ] Redacción de un artículo para animar a la comunidad a crear eventos sobre Rust 
- [ ] Localización de la página de la actividad en la web de hack rust
- [ ] Promoción en redes sociales

**Comment by nukeador** [comment]
Creo que las métricas quizá son demasiado optimistas 😛 

Las globales son:

```
Impact:

1,000 programmers/developers are introduced to and are well on their way to learning Rust
10 blog posts about events or other local Rust activities

Strength:

50 Mozillians organize Rust Hack and Learn events
50 new people signed up to Rust developer list
```

Deberíamos pensar a cuanta gente podemos llegar como comunidades hispanohablantes y de ahí cuantas personas podemos involucrar ;-)

**Comment by daviddagb2** [comment]
Hola Nuke, estas las agregué sin calcularlo bien, pero si estan bastante elevadas ¿Se podria habrir un hilo en Mozilla Hispano para que determinemos las metricas adecuadas? En el caso de los 20 mozillians talves se podrian dejar entre 5 a 10

**Comment by nukeador** [comment]
Claro, puedes abrir un tema como hice con el de Rust en el foro de participa

https://foro.mozilla-hispano.org/c/participa-en-el-proyecto

**Comment by nukeador** [comment]
¿Qué fechas podríamos estimar para tener los puntos definidos listos? ¿Es una semana (31 agosto) realista?

¿Qué otras personas podrían ayudarte aquí?

**Comment by daviddagb2** [comment]
Creo que para el viernes 2 de septiembre podriamos tener listo algo. adicionalmente he creado el hilo en el foro de Mozilla Hispano para definir mejor los puntos de esta actividad y definir los mismos:
[
https://foro.mozilla-hispano.org/t/actividad-dive-into-rust/22628](https://foro.mozilla-hispano.org/t/actividad-dive-into-rust/22628)

En caso de colaboaradores tenemos a Ivan Zeledon por el momento y veremos quien mas quieren colaborar

**Comment by nukeador** [comment]
@daviddagb2 no dudes en comentar por aquí si necesitas ayuda con algo.

**Comment by nukeador** [comment]
Hola,

¿Necesitáis ayuda para finalizar esto? Puedo ayudaros para publicar la localización de la web.

Creo que es interesante tenerlo antes de que la información quede obsoleta.

Un saludo.

**Comment by daviddagb2** [comment]
Hola @nukeador,  creo que si necesitariamos ayuda para finalizar y terminar de definir los objetivos. 
1. En algunos comentarios habiamos visto que era posible conseguir de 40 a 50 programadores a nivel hispano ¿que le parece al final esta Métrica para dejarla como definida?.
2. Puedo redactar un articulo o seria bueno que alguien con mas experiencia lo revise para publicarlo.
3.  Seria bueno tener alguna reunión con las personas interesadas en crear eventos relacioandos y empezar a promoverlos por diversos medios
4. Básicamente saber si los objetivos estan bien o si los cambiamos para finalizar con esta parte.

saludos

**Comment by nukeador** [comment]
> En algunos comentarios habiamos visto que era posible conseguir de 40 a 50 programadores a nivel hispano ¿que le parece al final esta Métrica para dejarla como definida?.

Aunque no es fácil es una buena meta, un reto realista.

> Puedo redactar un articulo o seria bueno que alguien con mas experiencia lo revise para publicarlo.

En [el foro de redacción](https://foro.mozilla-hispano.org/c/redaccion) hay mucha gente que puede ayudarte.

> Seria bueno tener alguna reunión con las personas interesadas en crear eventos relacioandos y empezar a promoverlos por diversos medios

Un ping a la gente que sabemos que está haciendo eventos de este tipo en el tema que ya abriste en el foro sería buena idea.

> Básicamente saber si los objetivos estan bien o si los cambiamos para finalizar con esta parte.

Yo les veo bien, no te preocupes mucho porque lo idea sería hacer un check de forma mensual y ver dónde estamos.

**Comment by daviddagb2** [comment]
Hola @nukeador, he dejado las metricas actuales a 30 desarrolladores, si vemos que se puede mejorar entonces lo aumentamos. Tambien para saber quienes estan colaborando u organizando eventos relacionados he creado el siguiente excel para asi anotar quienes pueden apoyar en esta actividad:

https://docs.google.com/spreadsheets/d/1VIAbeVW27i_qpZxAGbeCwApkBQfLE_KAiY_Cx27V7mk/edit?usp=sharing

**Comment by nukeador** [comment]
¿Necesitas ayuda para el blog post y la traducción? Podemos dar un toque a más gente para finalizarlo.

**Comment by daviddagb2** [comment]
si necesitamos ayuda con eso ¿podriamos conseguir alguien que nos ayude con el post?

**Comment by nukeador** [comment]
Intenta pregunta por el foro y telegram.

**Comment by gaby2300** [comment]
Yo puedo colaborar con la traducción

Saludos,
Gabriela

**Comment by yulireyji** [comment]
Hola! 

Apenas David me hizo favor de pasarme las ligas que no las encontraba :(

Yo puedo apoyar en la parte de los post de eventos e informativos.
